presentations/cargo/slides.adoc

```diff
@@ -96,7 +99,7 @@ For more details, check the http://doc.crates.io/manifest.html[docs]
 
 At the first build (or through `cargo update`), cargo calculates a version tree fulfilling the constraints stated in Cargo.toml. If successful, it will be saved in a Lockfile (Cargo.lock).
 
-It is recommendable to always keep the Lockfile under version control! (for repeatable builds)
+It can be considered to always keep the Lockfile under version control! (for repeatable builds)
```

Comment:
If you want this statement, I would reword as such:

> For repeatable builds, consider keeping the Lockfile under version control

However, this doesn't totally match what I would call "best practice", which would be:

* For binaries, include lock file (maybe delete directly in some nightly CI testing)
* For libraries, *don't* include a lock file

Reply:
The problem is that the best practice is debatable. If you don't include the lockfile, you lose all info about what your library was tested with during a test run.

Reply:
```suggestion
The Lockfile should always be kept under version control for applications, for libraries, it can be checked in to provide a repeatable test environment.
```

Reply:
@jamesmunns ^^ Better?